pages/events.html

```diff
@@ -6,6 +6,8 @@
 banner_file_mobile: banner--events-tour-map-sm.svg
 permalink: /events
 social_image: social-card--events.png
+social_image_alt_text: "Blue graphic reads 'Come meet us' with calendar graphics and USDS logo: navy blue shield with three stars, two stripes and yellow wings"
```

Comment:
Suggested text edit:

"Graphic with bold text that reads 'Come meet us.' Text is displayed next to an illustration of a calendar, and above the USDS logo, a navy blue shield with three stars, two stripes and yellow wings."